dictionary.json

```
@@ -348,6 +348,20 @@
     "examples": [],
     "fields": []
   },
+  {
+    "toaq": "ıq",
+    "type": "predicate",
+    "english": "▯ satisfies property ▯.",
```

Perhaps

```suggestion
    "english": "▯ satisfy property ▯.",
```

since it's nondistributive?